src/app/file-manager/transferables-grid/transferables-grid.component.ts

```diff
@@ -76,97 +192,76 @@ export class TransferablesGridComponent implements OnInit, AfterViewInit {
   applyFilter(filterValue: string) {
     filterValue = filterValue.trim();
     filterValue = filterValue.toLowerCase();
-    this.dataSource.filter = filterValue;
+    this.tabFilter = filterValue;
   }
 
   reset() {
-    this.store.dispatch(new Transferables.Reset());
+    // this.store.dispatch(new Transferables.Reset());
```

Comment:
If this code isn't needed, it should be deleted.

Reply:
This, and similar functions, are a sort of reminders to features to implement ... It provides some value as documentation and doesn't impact performance or code size. We would like to keep it.